### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by viaviaj**

### CODE REVIEW

**Review by fyarkiy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Internet Shop pet-project
```
Comment (dmytro-hryhoruk) [inline README.md]: don't call your project as "pet" this sound unprofessionally)

**Review by dmytro-hryhoruk — COMMENTED** [review]
let's also add set up guide for developers

**Comment by RomanDubovskyi** [comment]
what about setup guide for developers that Dima mentioned?

**Comment by viaviaj** [comment]
> what about setup guide for developers that Dima mentioned?
@RomanDubovskyi 
sorry, I didn't notice this comment
I fixed it, after HR'calls with Kate

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**